bc/subscription/templates/includes/search_htmx/search-form.html

```diff
@@ -0,0 +1,29 @@
+<form hx-ext="loading-states" hx-get="{% url 'search_cases' %}" hx-target="#searchResults" data-loading-states>
+  <div class="flex flex-col">
+    <div class="flex item-center">
+      <label for="simple-search" class="sr-only">Search</label>
+      <div class="relative w-full">
+        <div class="absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none text-gray-500">
+          {% include 'includes/inlines/magnifying-glass.svg'%}
+        </div>
+        <input type="text" id="simple-search" name="q"
+          class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full pl-10 p-2.5 placeholder-gray-400"
+          placeholder="i.e. https://www.courtlistener.com/docket/65895581/carroll-v-trump/" required>
```

Comment:
Let's use this one, which is about ending PACER fees: 
```suggestion
          placeholder="i.e. https://www.courtlistener.com/docket/4214664/national-veterans-legal-services-program-v-united-states/" required>
```